CONTRIBUTING.md

```diff
@@ -1 +1,8 @@
-We require contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+# Contribution guidelines
+
+We ***require*** contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+## Submitting Pull-Requests
+* Please make sure it builds : `make -C infer clean && make -C infer <target>`, refer to installation document for details
+* Add a test, and run the tests: `./scripts/test.sh` or `inferTest <target>` (i.e java)
```

Comment:
Thanks for adding this to the documentation. It is better to document only one way of running the tests and keep `./scripts/test.sh`. The script `inferTest` does not run the full stack of compilation before running the tests, and then does not check that everything is fine. This script is more intended to internal use and we may remove it from GitHub.